Cloning Schematize is run everytime I change the binning size

```bash
ubuntu@pantograph2:~/sh/test_pipeline/pipeline$ docker run -ti --rm --publish=3000:3000 --volume=`pwd`:/usr/src/app/data pipeline data/run1.B1phi1.i1.seqwish.gfa 10
### odgi build
### odgi sort
### odgi bin
### component segmentation
mkdir: cannot create directory ‘data/run1.B1phi1.i1.seqwish.seg’: File exists
fatal: destination path 'component_segmentation' already exists and is not an empty directory.
```

Can we prevent that?

No I get 
```
Failed to compile.

./src/PangenomeSchematic.js
Module not found: Can't resolve './data/data/run1.B1phi1.i1.seqwish.w1000.schematic.json' in '/usr/src/app/Schematize/src'

```

I guess one `data` too much in the code.

--- a/pipeline.sh
+++ b/pipeline.sh
@@ -94,7 +94,7 @@ if [ ! -d "Schematize" ]; then
   git clone --depth 1 https://github.com/graph-genome/Schematize
 fi
 cp ${SCHEMATICBIN} Schematize/src/data/
-sed -E "s|run1.B1phi1.i1.seqwish.w100.schematic.json|${SCHEMATICBIN}|g" Schematize/src/PangenomeSchematic.js > Schematize/src/PangenomeSchematic2.js
+sed -E "s|run1.B1phi1.i1.seqwish.w100.schematic.json|${SCHEMATICBIN:5}|g" Schematize/src/PangenomeSchematic.js > Schematize/src/PangenomeSchematic2.js
 mv Schematize/src/PangenomeSchematic2.js Schematize/src/PangenomeSchematic.js
 cd Schematize
 npm build

